wandb/sdk/interface/artifacts.py

```diff
@@ -108,13 +109,105 @@ def get_entries_in_directory(self, directory):
 
 
 class ArtifactEntry(object):
-    path: str
-    ref: Optional[str]
-    digest: str
-    birth_artifact_id: Optional[str]
-    size: Optional[int]
-    extra: Dict
-    local_path: Optional[str]
+    @property
+    def path(self) -> str:
+        """
+        The logical path of this artifact entry within the artifact.
```

Comment:
docstring style note: the "short-doc" should be on the same line as the first triple-quote.